rosdep/python.yaml

```
@@ -6768,6 +6768,33 @@ python3-gnupg:
   openembedded: [python3-gnupg@meta-python]
   opensuse: [python3-python-gnupg]
   ubuntu: [python3-gnupg]
+python3-google-auth:
+  debian: [python3-google-auth]
+  fedora: [python3-google-auth]
+  opensuse: [python3-google-auth]
+  rhel:
+    '*': [python3-google-auth]
+    '7': null
+  ubuntu: [python3-google-auth]
+python3-google-auth-httplib2:
+  debian:
+    '*': [python3-google-auth-httplib2]
+    buster: null
+  opensuse: [python3-google-auth-httplib2]
```

https://packages.fedoraproject.org/pkgs/google-auth-httplib2/python3-google-auth-httplib2/

```suggestion
  fedora: [python3-google-auth-httplib2]
  opensuse: [python3-google-auth-httplib2]
```